contracts/modules/cover/CoverUtilsLib.sol

```diff
@@ -13,6 +13,8 @@ import "./MinimalBeaconProxy.sol";
 
 library CoverUtilsLib {
 
+  uint private constant GLOBAL_CAPACITY_DENOMINATOR = 10_000;
```

Comment:
Since we have this in Cover, CoverUtilsLib, CoverViewere and StakingPool and it's the same value, wouldn't it be better to keep constants in a single library and import them from there?

Reply:
the issue i think here is that it'ss basically 1 or 2 common constants across those...

If you do a common lib i guess it helps with code tracking but you'd still do a full redeploy of all